backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java

```diff
@@ -35,8 +35,8 @@ void serialization() {
     TestResult r = TestResult.NEGATIVE;
     Date d = new Date();
 
-    Person p1 = _dataFactory.createMinimalPerson(o);
-    Person p2 = _dataFactory.createMinimalPerson(o);
+    Person p1 = _dataFactory.createFullPersonWithTelephone(o, "5168675309");
```

Comment:
Note that this change significantly increases the diff to this serialization test, but is necessary to get adequate coverage on the new fields. I've manually verified that these changes are as-expected